core/common/src/UtcOffset.kt

```diff
@@ -44,26 +51,26 @@ public expect class UtcOffset {
 }
 
 /**
- * Constructs a [UtcOffset].
+ * Constructs a [UtcOffset] from hours, minutes, and seconds components.
  *
  * All components must have the same sign.
  *
  * The bounds are checked: it is invalid to pass something other than `±[0; 59]` as the number of seconds or minutes.
- * For example, `UtcOffset(hours = 03, minutes = 61)` is invalid.
+ * For example, `UtcOffset(hours = 3, minutes = 61)` is invalid.
  *
- * However, if a component is the first non-null one, it can exceed these bounds.
- * So, for example, `UtcOffset(minutes = 241)` is valid.
+ * However, the first non-null component of highest order can exceed these bounds,
```

Comment:
> `the first non-null component of highest order`

Maybe just `the non-null component of the highest order` or `the first non-null component`?

Reply:
Chosen the former.